egs2/mediaspeech/asr1/local/download_and_untar.sh

```diff
@@ -0,0 +1 @@
+../../../../egs/commonvoice/asr1/local/download_and_untar.sh
```

Comment:
ditto

Reply:
This file is used to download the dataset